include/bluetooth/bluetooth.h

```diff
@@ -775,6 +827,60 @@ int bt_le_ext_adv_get_info(const struct bt_le_ext_adv *adv,
 typedef void bt_le_scan_cb_t(const bt_addr_le_t *addr, int8_t rssi,
 			     uint8_t adv_type, struct net_buf_simple *buf);
 
+/** @brief Set or update the periodic advertising parameters.
+ *
+ *  The periodic advertising parameters can only be set or updated on an
+ *  extended advertising set which is neither scannable, connectable or
+ *  anonymous.
+ *
+ *  @param adv   Advertising set object.
+ *  @param param Advertising parameters.
+ *
+ *  @return Zero on success or (negative) error code otherwise.
+ */
+int bt_le_per_adv_set_param(struct bt_le_ext_adv *adv,
+			    const struct bt_le_per_adv_param *param);
+
+/** @brief Set or update the periodic advertisement data.
+ *
+ *  The periodic advertisement data can only be set or updated on an
+ *  extended advertising set which is neither scannable, connectable or
+ *  anonymous.
+ *
+ *  @param adv       Advertising set object.
+ *  @param ad        Advertising data.
+ *  @param ad_len    Advertising data length.
+ *
+ *  @return          Zero on success or (negative) error code otherwise.
+ */
+
+int bt_le_per_adv_set_data(const struct bt_le_ext_adv *adv,
+			   const struct bt_data *ad, size_t ad_len);
+
+/** @brief Starts periodic advertising.
+ *
+ *  Enabling the periodic advertising can be done independently of extended
```

Comment:
Please add a note about the order in which those can be enabled, "independently in any order" perhaps?

Reply:
Added independent order :)